.github/workflows/release.yml

```diff
@@ -29,7 +29,6 @@ jobs:
       publish_docs: ${{ steps.check_commits.outputs.publish_docs }}
       commit_details: ${{ steps.check_commits.outputs.commit_details }}
       append_notes: ${{ steps.check_commits.outputs.append_notes }}
-      pr_description: ${{ steps.get_pr_description.outputs.pr_description }}
```

Comment:
- The PR Description was removed, but the corresponding display in the `create-confluence-page` job was not fully removed. Ensure there are no unintended dependencies.